src/main/java/greedy/greedybot/application/matching/dto/MatchingResultAnnouncement.java

```diff
@@ -0,0 +1,26 @@
+package greedy.greedybot.application.matching.dto;
+
+import greedy.greedybot.application.matching.MatchingResult;
+
+import java.util.List;
+import java.util.Map;
+
+public record MatchingResultAnnouncement(
+        String announcement
+) {
+    public static MatchingResultAnnouncement of(final MatchingResult matchingResult) {
+        final StringBuilder stringBuilder = new StringBuilder();
+        stringBuilder.append("[리뷰이]  --  [리뷰어]\n");
+
+        for (Map.Entry<String, List<String>> entry : matchingResult.getResult().entrySet()) {
+            String reviewer = entry.getKey();
+            for (String reviewee : entry.getValue()) {
+                stringBuilder.append(reviewee)
+                        .append("    ->   ")
+                        .append(reviewer)
+                        .append("\n");
+            }
+        }
```

Comment:
```suggestion
        final Map<String, List<String>> revieweesByReviewer = matchingResult.getResult();
        for (final String reviewer : revieweesByReviewer.keySet()) {
            final List<String> reviewees = revieweesByReviewer.get(reviewer);
            reviewees.forEach(reviewee -> stringBuilder.append(reviewee)
                    .append("    ->   ")
                    .append(reviewer)
                    .append("\n"));
        }
```
취향차이긴 한데, 이렇게 depth를 줄여 볼 수 있을 것 같아요.

Reply:
Depth를 줄이는 것이 코드 읽기에 더 좋을거 같아 적용하겠습니다!